x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -40,18 +40,24 @@ export interface AddModelFlyoutProps {
   modelDownloads: ModelItem[];
   onClose: () => void;
   onSubmit: (modelId: string) => void;
+  'data-test-subj'?: string;
 }
 
-type FlyoutTabId = 'clickToDownload' | 'manualDownload';
+export type AddModelFlyoutTabId = 'clickToDownload' | 'manualDownload';
 
 /**
  * Flyout for downloading elastic curated models and showing instructions for importing third-party models.
  */
-export const AddModelFlyout: FC<AddModelFlyoutProps> = ({ onClose, onSubmit, modelDownloads }) => {
+export const AddModelFlyout: FC<AddModelFlyoutProps> = ({
+  onClose,
+  onSubmit,
+  modelDownloads,
+  'data-test-subj': dataTestSubj,
```

Comment:
@darnautov this seems odd, Intellij reports it as unused.

Reply:
yes, because you do not pass it to `EuiFlyout`. You have a static string `mlAddTrainedModelFlyout` there. But also you do not pass the data test subject to the `AddModelFlyout` component as well. You have two options: 
1. Pass `data-test-subj` to the `AddModelFlyout` component and then internally to `EuiFlyout`. In that case, you can set data test subj to EUI flyout dynamically from the parent component of `AddModelFlyout` 
2. Just have a static `data-test-subj` for `EuiFlyout` as it is now, and remove the `data-test-subj` prop from `AddModelFlyout` altogether. 

I reckon the 2nd option suffice.